extensions/src/ACRE2Mumble/MumbleCallbacks_static.cpp

```diff
@@ -29,3 +29,29 @@ mumble_version_t mumble_getAPIVersion() {
 mumble_version_t mumble_getVersion() {
     return mumble_version_t{ACRE_VERSION_MAJOR, ACRE_VERSION_MINOR, ACRE_VERSION_SUBMINOR};
 }
+
+struct MumbleStringWrapper mumble_getAuthor() {
+    static const char* author = "ACRE2 developers";
+
+    MumbleStringWrapper wrapper;
+    wrapper.data = author;
+    wrapper.size = strlen(author);
+    wrapper.needsReleasing = false;
+
+    return wrapper;
+}
+
+struct MumbleStringWrapper mumble_getDescription() {
+    static const char* description = "ACRE2 is a full fledged communications suite for Arma 3, allowing realistic radio and voice communications.";
```

Comment:
Should return `ACRE_DESC` as in TeamSpeak.
https://github.com/IDI-Systems/acre2/blob/a0572a4dd4fe871b8b896e544300605f4cd0d87c/extensions/src/ACRE2TS/TSCallbacks_init.cpp#L46